libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts

```diff
@@ -130,6 +131,11 @@ export class ExecuteBridgeRequest {
         afterResponse:
           command.afterResponse !== undefined ? [command.afterResponse] : [],
       },
+      /*
+       * Reject self-signed and invalid certificates in Production environments but allow them in Development
+       * as it's common for developers to use self-signed certificates in local environments.
+       */
+      rejectUnauthorized: true,
```

Comment:
@SokratisVidros the new error response looks great! One thing though - you forgot to revert the change here to check Environement.name after testing

Reply:
😮‍💨 Fixed on next. Thanks for noticing. That's what happens if you are committing code in between Duplo gaming sessions.